packages/core/test/actions.test.ts

```diff
@@ -1354,96 +1475,6 @@ describe('choose', () => {
 
     expect(service.state.context).toEqual({ answer: 42 });
   });
-
-  // https://github.com/davidkpiano/xstate/issues/1109
-  it('exit actions should be called when invoked machine reaches final state', (done) => {
```

Comment:
I've moved those to the new describe blocks as those were nested in the `"choose"` block which didn't quite make sense for those tests.